projects/1.21/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -0,0 +1,2009 @@
+  "ars_nouveau.glyph_name.glyph_touch": "触摸",
+  "ars_nouveau.glyph_name.glyph_underfoot": "脚下",
+  "ars_nouveau.glyph_name.glyph_wall": "魔墙",
+  "ars_nouveau.glyph_name.glyph_wind_burst": "风爆",
+  "ars_nouveau.glyph_name.glyph_wind_shear": "风切",
+  "ars_nouveau.glyph_name.glyph_wither": "凋零",
+  "ars_nouveau.glyph_name.reset": "重置",
+  "ars_nouveau.glyph_name.rewind": "回溯",
+  "ars_nouveau.glyph_name.wololo": "Wololo",
+  "ars_nouveau.glyph_of": "%s之魔符",
+  "ars_nouveau.gui.color": "颜色选择器",
+  "ars_nouveau.gui.discord": "加入Discord服务器，法术灵感、全新更新、社区支持应有尽有！",
+  "ars_nouveau.gui.familiar": "使魔",
+  "ars_nouveau.gui.notebook": "说明文档",
+  "ars_nouveau.gui.settings": "设置与奖励",
+  "ars_nouveau.gui.sounds": "声音",
+  "ars_nouveau.happiness": "幸福度",
+  "ars_nouveau.heat": "热量",
+  "ars_nouveau.home_set": "家园已设置。",
+  "ars_nouveau.ignore_set": "星宝石兽会忽略这些物品。",
+  "ars_nouveau.imbuement.crafting_started": "正在合成%s。提供魔源能增加合成速度。",
+  "ars_nouveau.imbuement.norecipe": "未找到配方。合成需要的基座必须放置在距灌注室1格范围内。",
+  "ars_nouveau.imbuement.use_apparatus": "你是否希望使用附魔装置执行此配方？",
+  "ars_nouveau.important": "重要事项",
+  "ars_nouveau.invalid_lectern": "连接了无效的讲台。",
+  "ars_nouveau.invalid_spell": "无效法术",
+  "ars_nouveau.item_crafting": "物品合成",
+  "ars_nouveau.item_crafting_setting": "多重物品合成",
+  "ars_nouveau.item_detector.connected": "正在检索X：%s Y：%s Z：%s处的物品",
+  "ars_nouveau.item_detector.count": "信号阈值：%s",
+  "ars_nouveau.item_detector.inverted": "信号反转：%s",
+  "ars_nouveau.item_detector.item": "%s",
+  "ars_nouveau.item_detector.powered": "激活：%s",
+  "ars_nouveau.item_transport": "物品运输",
+  "ars_nouveau.keyword.familiar": "使魔",
+  "ars_nouveau.keyword.glyph": "魔符",
+  "ars_nouveau.keyword.ritual": "仪式",
+  "ars_nouveau.keyword.thread": "纤维",
+  "ars_nouveau.lectern_blacklist": "此方块已被禁止连接至存储讲台。",
+  "ars_nouveau.lectern_out_of_range": "讲台超出范围。",
+  "ars_nouveau.level": "等级",
+  "ars_nouveau.levels_required": "需要等级：%s",
+  "ars_nouveau.light_message": "新生魔艺自带动态光源。可使用“/ars-light on”命令开启。此消息不会再次出现！",
+  "ars_nouveau.lights_off": "你已关闭动态光源。",
+  "ars_nouveau.lights_on": "你已开启动态光源。CPU较弱或RAM分配较少的用户可能会出现卡顿，装有Optifine等“性能优化”模组时还可能会出现未知问题。再次运行命令或修改ars_nouveau-client配置以禁用。",
+  "ars_nouveau.lily": "Patreon赞助者可以召唤Lily——尾巴摇摇的忠诚伙伴。",
+  "ars_nouveau.locked": "已锁定",
+  "ars_nouveau.machines": "机器",
+  "ars_nouveau.machines_desc": "魔法机器",
+  "ars_nouveau.mana_cost.high": "高",
+  "ars_nouveau.mana_cost.low": "低",
+  "ars_nouveau.mana_cost.medium": "中",
+  "ars_nouveau.mana_cost.none": "无",
+  "ars_nouveau.manipulation_essence.tooltip": "可抄入法术羊皮纸或施法工具，以永久隐藏法术。",
+  "ars_nouveau.melder.destination_invalid": "目标无法接受当前的混合药水。",
+  "ars_nouveau.melder.from_capped": "混合器只能混合两个药水罐。潜行时使用支配之杖可清空其药水罐配置。",
+  "ars_nouveau.melder.from_set": "输入药水罐已设置%s/2。",
+  "ars_nouveau.melder.needs_potion": "相连的药水罐需要药水。",
+  "ars_nouveau.melder.no_to_pos": "需对目标药水罐使用支配之杖进行连接。",
+  "ars_nouveau.melder.output_duplicate_effect": "无法混合同效果不同等级的药水。",
+  "ars_nouveau.melder.output_not_unique": "输出的药水不具独立性。",

```

看了[源代码](https://github.com/baileyholl/Ars-Nouveau/blob/main/src/main/java/com/hollingsworth/arsnouveau/common/block/tile/PotionMelderTile.java#L171)
这里是检测混合后的药水是否和混合前一模，所以不应该翻译成独立性，或许是：
```suggestion
  "ars_nouveau.melder.output_not_unique": "输出的药水不是独特混合。",

```

---

我也看过，但作者的描述实在太难理解，甚至不如直接说不允许产物与输入一致

---

换成“输出的药水不可与输入一致”

---

可以，这里原文语义不清也没必要按照原文来了，翻译还是让人看懂吧。